lib/Service/ContextService.php

```diff
@@ -207,6 +207,12 @@ public function update(int $contextId, string $userId, ?string $name, ?string $i
 			unset($nodesBeingRemoved);
 
 			foreach ($nodesBeingAdded as $node) {
+				$nodeType = (int)($node['type']);
+				$nodeId = (int)($node['id']);
+				if (!$this->permissionsService->canManageNodeById($nodeType, $nodeId, $userId)) {
+					throw new PermissionError(sprintf('Owner cannot manage node %d (type %d)', $nodeId, $nodeType));
```

Comment:
I don't know why, but this error message does not print into the logs

Reply:
PermissionErrors are not caught in https://github.com/nextcloud/tables/blob/main/lib/Controller/ContextController.php#L149C17-L151 that would  need to be added